Curs/src/views/evaluations/evaluation.js

```diff
@@ -0,0 +1,92 @@
+interviewApp.EvaluationsPage = function(options = {} )  {
+
+    const headings = [
+        'Nume','Tehnologie','Nivel', 'Details'
+    ]
+
+    const rows = JSON.parse(localStorage.getItem("storageKey"))
```

Comment:
Same note as above about potentially unsafe `JSON.parse` operation.

Reply:
![test](https://user-images.githubusercontent.com/31060543/30113211-99d21d74-931c-11e7-993b-0d4e6eda05cb.png)

It is ok like that?